.changeset/cyan-chefs-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/explorer": patch
+---
+
+Dozer integration has been added for Redstone and Garnet. When the Explorer is run on Anvil, the local indexer is used. For Garnet or Redstone, the indexed data is retrieved from a remotely hosted Dozer instance.
```

Comment:
I don't think end users know/care what Dozer is and we've started moving towards calling this SQL API (and could say "hosted SQL API" or "Redstone SQL API" or similar when referring to the hosted variant).